cvmfs/catalog_sql.cc

```diff
@@ -1101,6 +1101,58 @@ bool SqlDirentUpdate::BindDirent(const DirectoryEntry &entry) {
 
 //------------------------------------------------------------------------------
 
+SqlDirentRename::SqlDirentRename(const CatalogDatabase &database) {
```

Comment:
Could be better to utilize SqlDirentUpdate combination and SQLDirentRename and remove metadata update from this query